energy-growers/src/app/news-gallery-thumbs/page.tsx

```diff
@@ -0,0 +1,20 @@
+"use client"
+
+import React from 'react';
+import NewsGalleryThumbs from '../../components/NewsGalleryThumbs/NewsGalleryThumbs';
+import product from "../../../public/image/newsgallerythumbs.png";
+
+const Page = () => {
+    return (
+      <div>
+        <NewsGalleryThumbs
+          image={product}
+          title="Categoria / Categoria"
+          link="/"
+          description="Titolo della news fino ad un massimo di sei righe di testo. Titolo della news fino ad un massimo di sei righe di testo."
+        />
```

Comment:
```suggestion
 <NewsGalleryThumbs
        id=""
        createdAt={"" as unknown as Date}
        updatedAt={"" as unknown as Date}
        priority={1}
        key={2}
        image={product as unknown as EgImage}
        category="Categoria / Categoria"
        newsButton={{ link: { label: " ", url: "" } }}
        title="Titolo della news fino ad un massimo di sei righe di testo. Titolo della news fino ad un massimo di sei righe di testo."
      />
```

dobbiamo cominciare a usare le interfacce fatte da samuel per una prossima integrazione con il back-end